independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -7,7 +7,7 @@
     "contact": {
       "name":"Chandra Sekhar & Harshini Varma",
       "url":"https://github.com/sekharmalla",
-      "email":"Sekharmalla.vfx@gmail.com"
+      "email":"chandra.malla@msftcommunity.com"
```

Comment:
@sekharmalla please move changes for other connectors to separate PRs.

Reply:
@v-EgorKozhadei 
Please do let me know if I can delete this file or how should I move this to a new PR. 

Thanks